softwarereview_editor.Rmd

```diff
@@ -126,48 +121,54 @@ Each submission should be reviewed by *two* package reviewers. Although it is fi
 
-- On assuming EiC rotation, reviews status of current open reviews thanks to [devguider](#eic-report) and reminds editors to respond or update status as needed.
+- Responding to issues posted to [the software-review-meta repo
 
-- Responds to issues posted to the software-review-meta repo
-
-- Makes decisions on scope/overlap for pre-submission inquiries, referrals from JOSS or other publication partners, and submissions if they see an ambiguous case (This last case may also be done by handling editors (see below)). To initiate discussion, this is posted to the rOpenSci Slack editors-only channel along with a small summary of what the (pre-)submitted/referred submission is about, what doubts the EiC has i.e. digesting information a bit. If after one day or two the EiC feels they haven't received enough answers, they can ping all editors.
+- Making decisions on scope and overlap for pre-submission inquiries, referrals from JOSS or other publication partners, and submissions. Discussions should be initiated in the rOpenSci Slack editors-only channel through summarising the (pre-)submitted/referred software, along with any concerns the EiC might have. If after the EiC feels they haven't received enough answers after a day or two, they can ping all editors.
   
   - Any editor should feel free to step in on these. See [this section](#outofscoperesponse) about how to respond to out-of-scope (pre-) submissions.
-  - After explaining the out-of-scope decision, write an issue comment `@ropensci-review-bot out-of-scope`.
-
-- Requests a new EiC when their rotation is up (set a calendar reminder ahead of your expected end date and ask for volunteers in the editors' Slack channel)
+  - After explaining an out-of-scope decision, write an issue comment `@ropensci-review-bot out-of-scope`.
 
-### Using `devguider::devguide_eic_report()` {#eic-report}
+### The rOpenSci Editorial Dashboard {#eic-dashboard}
 
-Install devguider and run `devguider::devguide_eic_report()`, open the HTML report in a browser.
+The *rOpenSci Editorial Dashboard* is updated daily, primary by extracting information on all software review issues on GitHub, along with additional information from Slack and our Airtable database.
+The dashboard provides an up-to-date overview of our editorial team, their recent reponsibilities, and the current state of all software review issues.
+The EiC (or any editors who are interested) can gain an overview of the editorial team status, availability, and recent workloads on [the *editors* page](https://ropensci-review-tools.github.io/dashboard/editors.html).
+This should be used to find and assign editors for new software review issues.
+An overview of all current software reviews is on [the **Software Review* page](https://ropensci-review-tools.github.io/dashboard/reviews.html).
```

Comment:
do we want to create a better link? a subdomain of ropensci dot org?

Reply:
But this is in the dashboard sub-section, and so links to the dashboard page that that sentence is all about. Or have I misunderstood something?

Reply:
I mean, should the dashboard live at a better URL

Reply:
Oh that. Noam and I did once briefly discuss that, but were both okay for now with it there. But better ideas from you would be great ...?

Reply:
I'd really be in favor of an ropensci.org subdomain.

Reply:
Would that work smoothly with the hugo site? If you want to set that up, it's be great

Reply:
@mpadge now that the subdomain exists, could we finalize this PR? :smile_cat:

Reply:
> @mpadge now that the subdomain exists, could we finalize this PR? 😸

Yes indeed; I'll get on to it today. Thanks for the ping